### PR TITLE
Allow multiple anonymous rules in `no-identical-title`

### PR DESCRIPTION
**Pull request by kevva**
Being `undefined`, I guess we can assume it's a anonymous title.

### CODE REVIEW

File: rules/no-identical-title.js [hunk 1]
```diff
@@ -39,7 +38,7 @@ module.exports = function (context) {
 			var args = node.arguments;
 			var titleNode = args.length > 1 || ava.hasTestModifier('todo') ? args[0] : undefined;
 
-			if (!isStatic(titleNode)) {
+			if (titleNode === undefined || !isStatic(titleNode)) {
```
Comment (kevva) [inline rules/no-identical-title.js]: Checking explicitly here so we don't treat empty strings or whatever as anonymous tests.

File: test/no-identical-title.js [hunk 2]
```diff
@@ -29,6 +29,8 @@ test(() => {
 			header + 'test.beforeEach(t => {}); test.beforeEach(t => {});',
 			header + 'test.afterEach(t => {}); test.afterEach(t => {});',
 			header + 'test.cb.before(t => {}); test.before.cb(t => {});',
+			header + 'test(t => {}); test(t => {});',
```
Comment (jamestalmage) [inline test/no-identical-title.js]: Maybe add a comment here - "multiple anonymous tests covered by the if-multiple rule"

File: rules/no-identical-title.js (hunk 1 above)
Comment (jamestalmage) [inline rules/no-identical-title.js]: Another comment here:

> Don't flag computed titles or anonymous tests (anon tests covered in the if-multiple rule)

File: test/no-identical-title.js (hunk 2 above)
Comment (kevva) [inline test/no-identical-title.js]: Good idea.

**Comment by jamestalmage** [comment]
LGTM

**Comment by sindresorhus** [comment]
LGTM2

Thanks @kevva :)

**Pull request merged**